tools/wheels/LICENSE_win32.txt

```diff
@@ -880,7 +880,7 @@ Public License instead of this License.  But first, please read
 <http://www.gnu.org/philosophy/why-not-lgpl.html>.
 
 Name: libquadmath
-Files: numpy\.libs\libopenb*.dll
+Files: numpy.libs\libopenb*.dll
```

Comment:
Are we statically linking a `.dll`? Or should it be `.a` for Windows? And we're sure it should be an OpenBLAS library name for quadmath instead of a quadmath-specific lib? Is it an indirect link to quadmath via OpenBLAS then?

Reply:
I guess that's what's in the wheel, although maybe the static linking line below this line is less clear to me via DLL.

Reply:
I think this is correct. The one file that is vendored is `libopenblas64*.dll`, and that DLL also contains statically linked `libgcc` and `libquadmath`.